htdocs/xoops_lib/modules/protector/filters_disabled/postcommon_register_insert_js_check.php

```
@@ -12,7 +12,7 @@ public function execute()
     {
         ob_start([$this, 'ob_filter']);
 
-        if (!empty($_POST)) {
+        if ($_SERVER['REQUEST_METHOD'] === 'POST' && \Xmf\Request::hasVar('antispam_md5', 'POST')) {
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Xmf\Request' in method 'execute'. (undefined)

(StaticAccess)